weg-li/UI/Components/SubmitButton.swift

```diff
@@ -70,23 +70,35 @@ struct SubmitButton: View {
         case readyToSubmit(ordnungsamt: String)
     }
     
-    @ViewBuilder var body: some View {
-        if case .readyToSubmit(let ordnungsamt) = state {
-            return Button(action: action) { ReadySubmitButton(ordnungsamt: ordnungsamt) }
-        } else if case .unsupportedLocation = state {
-            return Button(action: action) { UnsupportedLocationButton() }
-        } else {
-            return Button(action: action) { MissingDataButton() }
+     var body: some View {
+        Button(action: action) { () -> AnyView in
```

Comment:
I don't see the reason for `AnyView` here because the return object is always a `Button`?

Reply:
The name is misleading, they are actually Views.